library/globals.inc.php

```diff
@@ -172,6 +172,37 @@ function gblTimeZones()
             xl('Pick a general theme (need to logout/login after change this setting).')
         ),
 
+        'font-family' => [
+            xl('Default font'),
+            [
+                '__default__' => 'Use Theme Font',
+                'Arial, Helvetica, sans-serif' => "Arial",
+                '"Arial Black", Gadget, sans-serif' => "Arial Black",
```

Comment:
hmm actually, not sure what this is. For some reason, you have " around some terms and not others?
I am guessing this is needed for the css. If so, then disregard.